model/hiscore.go

```diff
@@ -29,41 +29,86 @@ const (
 )
 
 const (
-	Overall       = 0
-	Attack        = 1
-	Defence       = 2
-	Strength      = 3
-	Hitpoints     = 4
-	Ranged        = 5
-	Prayer        = 6
-	Magic         = 7
-	Cooking       = 8
-	Woodcutting   = 9
-	Fletching     = 10
-	Fishing       = 11
-	Firemaking    = 12
-	Crafting      = 13
-	Smithing      = 14
-	Mining        = 15
-	Herblore      = 16
-	Agility       = 17
-	Thieving      = 18
-	Slayer        = 19
-	Farming       = 20
-	Runecraft     = 21
-	Hunter        = 22
-	Construction  = 23
-	Bhhunter      = 24
-	Bhrogue       = 25
-	Lmsrank       = 26
-	Cluesall      = 27
-	Cluesbeginner = 28
-	Clueseasy     = 29
-	Cluesmedium   = 30
-	Clueshard     = 31
-	Clueselite    = 32
-	Cluesmaster   = 33
-	Offset        = Bhhunter
+	Overall               = 0
```

Comment:
```suggestion
	Overall               = iota
	Attack
	Strength
	...
```

lets get rid of this awfulness and use this instead

Reply:
Will take care of it tomorrow when I'm not hammered 👍